availability.py

```
@@ -0,0 +1,488 @@
+
+class AvailabilityStatsRequestSpec(BaseModel):
+    from_date: datetime.date
+    to_date: datetime.date
+    resource_type: SchedulableResourceTypeOptions
+    resource_id: UUID4
+
+    @model_validator(mode="after")
+    def validate_period(self):
+        max_period = 32
+        if self.from_date > self.to_date:
+            raise ValidationError("From Date cannot be after To Date")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)